### PR TITLE
refactor(signing): switch xpi and stage variables to new naming fo…

### PR DESCRIPTION
**Pull request by ahal**
…rmat

For consistency, I've made the following changes:

1. Credentials for Autograph stage now go into variables prefixed with `AUTOGRAPH_STAGE`
2. Credentials for privileged xpi signing, now goes into `XPI_PRIVILEGED` variables

### CODE REVIEW

File: signingscript/docker.d/passwords.yml [hunk 1]
```diff
@@ -201,8 +201,8 @@ in:
              ["autograph_gpg"]
             ]
           - ["https://autograph-external.stage.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_XPI_STAGE_USERNAME"},
-             {"$eval": "AUTOGRAPH_XPI_STAGE_PASSWORD"},
+             {"$eval": "AUTOGRAPH_STAGE_XPI_PRIVILEGED_USERNAME"},
+             {"$eval": "AUTOGRAPH_STAGE_XPI_PRIVILEGED_PASSWORD"},
```
Comment (jcristau) [inline signingscript/docker.d/passwords.yml]: This is non-privileged signing, should it switch variables anyway?

File: signingscript/docker.d/init_worker.sh [hunk 2]
```diff
@@ -160,9 +160,9 @@ case $ENV in
...
         test_var_set 'AUTOGRAPH_AUTHENTICODE_SHA2_USERNAME'
         test_var_set 'AUTOGRAPH_MAR_PASSWORD'
-        test_var_set 'AUTOGRAPH_MAR_STAGE_PASSWORD'
-        test_var_set 'AUTOGRAPH_MAR_STAGE_USERNAME'
         test_var_set 'AUTOGRAPH_MAR_USERNAME'
+        test_var_set 'AUTOGRAPH_STAGE_MAR_PASSWORD'
+        test_var_set 'AUTOGRAPH_STAGE_MAR_USERNAME'
         test_var_set 'AUTOGRAPH_GPG_PASSWORD'
         test_var_set 'AUTOGRAPH_GPG_USERNAME'
         test_var_set 'GPG_PUBKEY_PATH'
```
Comment (jcristau) [inline signingscript/docker.d/init_worker.sh:170]: Might want to take the opportunity to add the xpi signing variables here, that I forgot when adding xpi stage signing to  adhoc-t.

File: signingscript/docker.d/passwords.yml (hunk 1 above)
Comment (ahal) [reply signingscript/docker.d/passwords.yml]: Good catch, I thought it was privileged but didn't notice the `cas_cur_extension` vs `ca_cur_webextension`

File: signingscript/docker.d/init_worker.sh [hunk 3]
```diff
@@ -261,6 +261,8 @@ case $ENV in
         test_var_set 'AUTOGRAPH_AUTHENTICODE_EV_USERNAME'
         test_var_set 'AUTOGRAPH_MAR_RELEASE_PASSWORD'
         test_var_set 'AUTOGRAPH_MAR_RELEASE_USERNAME'
+        test_var_set 'AUTOGRAPH_STAGE_XPI_PASSWORD'
+        test_var_set 'AUTOGRAPH_STAGE_XPI_USERNAME'
```
Comment (jcristau) [inline signingscript/docker.d/init_worker.sh]: I think these should be in dev/fake-prod, not prod?

Comment (ahal) [reply signingscript/docker.d/init_worker.sh]: Sigh, thanks.. Good to see the test caught it at least!

**Review by jcristau — APPROVED** [review]

**Pull request merged**